deploy.py

```diff
@@ -59,6 +59,8 @@ def build_config_and_deploy(stage):
     print(''.join(['Writing: ', filename]))
     with open(filename, 'w') as config_file:
         config_file.write(json.dumps(CONFIG_BASE))
+    # export poetry into requirements
+    subprocess.call(['poetry', 'export', '-f', 'requirements.txt', '>', 'requirements.txt'])
```

Comment:
I just have to say this is bizarre syntax. I looked it up in the doc and it seems right, though. I guess the `requirements.txt` after the `-f` is not a filename but a format name keyword. It seems to me that they should have just called it requirements. Oh well. Anyway, this is fine and what the doc seems to recommend.

Reply:
It is strange isn't it. I'm worried that this change will completely break the deployment of the chalice application because it relies on `requirements.txt`. This is my workaround that I can't exactly test unless I try to deploy over it. Any thoughts on what to best do are appreciated.